iree/compiler/Codegen/LLVMCPU/LLVMCPUTileAndVectorizeLinalgTensorOps.cpp

```diff
@@ -170,10 +170,9 @@ void LLVMCPUTileAndVectorizePass::runOnOperation() {
 
   // Op specific conversion.
   {
-    RewritePatternSet vectorizeOpsPattenrs(context);
-    populateLinalgToVectorVectorizeMMT4dPatterns(context, vectorizeOpsPattenrs);
-    if (failed(applyPatternsAndFoldGreedily(funcOp,
-                                            std::move(vectorizeOpsPattenrs)))) {
+    RewritePatternSet patterns(context);
```

Comment:
I see this pattern used in two places. We need to decide which path we are going to take for MMT4D. Lot of this is kept around based on decision made here. Ideally we need this along only a single pipeline.

Reply:
Right, I updated this PR to only use the newer LLVMCPUTileFustAndVectorizeLinalgTensorOps.cpp:

```diff
--- a/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp
+++ b/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp
@@ -87,12 +87,9 @@ static Optional<llvm::Triple> getTargetTriple(FuncOp entryPointFn) {
 static DispatchLoweringPassPipeline getDispatchLoweringPassPipeline(
     FuncOp entryPointFn, Operation *op) {
   return TypeSwitch<Operation *, DispatchLoweringPassPipeline>(op)
-      .Case<linalg::ContractionOpInterface>([&](auto op) {
+      .Case<linalg::ContractionOpInterface, linalg::Mmt4DOp>([&](auto op) {
         return DispatchLoweringPassPipeline::CPUTileFuseAndVectorize;
       })
-      .Case<linalg::Mmt4DOp>([&](auto op) {
-        return DispatchLoweringPassPipeline::CPUTensorToVectors;
-      })
```

and I dropped the changes to the old pipeline.